pkg/reconciler/pipelinerun/cancel.go

```diff
@@ -122,6 +127,42 @@ func cancelPipelineTaskRuns(ctx context.Context, logger *zap.SugaredLogger, pr *
 	return errs
 }
 
+// filterChildObjectsFromPRStatus returns taskruns and runs owned by the pipelineRun, based on the value of the embedded status flag.
+func filterChildObjectsFromPRStatus(ctx context.Context, prs v1beta1.PipelineRunStatus) ([]string, []string, error) {
```

Comment:
I think there is some confusion between this function and `filterTaskRunsForPipelineRun` (and likewise for runs). this function doesn't appear to check owner references at all.

Reply:
Yeah, lemme clarify.